webdriver/tests/bidi/browsing_context/__init__.py

```diff
@@ -11,7 +11,7 @@
 
 
 def assert_browsing_context(
-    info, context, children=None, is_root=True, parent=None, url=None
+    info, context, children=None, is_root=True, parent=None, url=None, user_context = 'default'
```

Comment:
With this change all the tests using this assertion will fail if the browser hasn't implemented that feature yet. Could we have this optionally for now and only checked in those tests where it is needed?

Reply:
Sure!